packages/postgres-database/src/simcore_postgres_database/models/comp_run_snapshot_tasks.py

```diff
@@ -0,0 +1,129 @@
+"""Computational Tasks Table"""
+
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+from sqlalchemy.dialects.postgresql import ENUM
+
+from ._common import (
+    RefActions,
+    column_created_datetime,
+    column_modified_datetime,
+)
+from .base import metadata
+from .comp_pipeline import StateType
+from .comp_runs import comp_runs
+
+comp_run_snapshot_tasks = sa.Table(
+    "comp_run_snapshot_tasks",
+    metadata,
+    sa.Column(
+        "snapshot_task_id",
+        sa.Integer,
+        primary_key=True,
+    ),
+    sa.Column(
+        "run_id",
+        sa.Integer,
+        sa.ForeignKey(
+            comp_runs.c.run_id,
+            name="fk_snapshot_tasks_to_comp_runs",
+            onupdate=RefActions.CASCADE,
+            ondelete=RefActions.CASCADE,
+        ),
+        nullable=False,
+    ),
+    sa.Column(
+        "project_id",
+        sa.String,
+        doc="Project that contains the node associated to this task",
+    ),
+    sa.Column("node_id", sa.String, doc="Node associated to this task"),
+    sa.Column(
+        "node_class",
+        ENUM(
```

Comment:
should this not be the very same enum as in the other table?

Reply:
just read the comment. ok

Reply:
please check the comment below. maybe that is your problem there.
better to manually fix the alembic migration than to have weird handling of enums I think.